vendor/assets/stylesheets/dvl/core/includes.scss

```diff
@@ -141,3 +141,9 @@ $easeInOutBack:  cubic-bezier(0.680, -0.550, 0.265, 1.550);
   white-space: nowrap;
   text-overflow: ellipsis;
 }
+
+@mixin word_break {
```

Comment:
Selector `dropdown_toggle_button` should be written in lowercase with hyphens<br>Name of mixin `word_break` should be written in all lowercase letters with hyphens instead of underscores